### PR TITLE
ADD: Multisig Vaults Help Section

### PR DESCRIPTION
**Pull request by ncoelho**

### CODE REVIEW

**Comment by marcosrdz** [comment]
@ncoelho  deepscan

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mehuzhnxn8380451j4dqe75jmc

[download apk](http://filestorage.bluewallet.io:1488/1607530017.apk)

File: screen/wallets/addMultisigHelp.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  ...BlueNavigationStyle(),
+  title: '',
+  gestureEnabled: false,
+  swipeEnabled: false,
+  headerStyle: {
+    backgroundColor: '#0070FF',
+    borderBottomWidth: 0,
+    borderBottomColor: '#0070FF',
+    elevation: 0,
+    // shadowRadius: 0,
```
Comment (Overtorment) [inline screen/wallets/addMultisigHelp.js:132]: dead code typically should not be in or, but ok, can cleanup later

**Review by Overtorment — APPROVED** [review]

**Comment by Overtorment** [comment]
lgtm. 
btw, maybe there should also be a success screen, when ms is created..?

File: screen/wallets/addMultisigHelp.js (hunk 1 above)
Comment (ncoelho) [reply screen/wallets/addMultisigHelp.js:132]: Which part is the dead code?
This is custom styling the header

Comment (ncoelho) [reply screen/wallets/addMultisigHelp.js:132]: The comment part right?

Comment (Overtorment) [reply screen/wallets/addMultisigHelp.js:132]: yep. commented-out code is dead code

**Comment by ncoelho** [comment]
> lgtm.
> btw, maybe there should also be a success screen, when ms is created..?

We could improve the wallet animation when it loads to make the success more obvious, but no rush on that.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xwbaaxry2f6p1jg445fajtpvjm

[download apk](http://filestorage.bluewallet.io:1488/1607536291.apk)

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**